resources/xlf/en/big-data-cluster.xlf

```diff
@@ -24,7 +24,7 @@
   </body></file>
   <file original="extensions/big-data-cluster/out/bigDataCluster/tree/addControllerTreeNode" source-language="en" datatype="plaintext"><body>
     <trans-unit id="textBigDataClusterControllerWithDots">
-      <source xml:lang="en">Add Big Data Cluster Controller...</source>
+      <source xml:lang="en">Add big data cluster Controller...</source>
```

Comment:
You lowercased controller earlier - did you want to do that here too?

Reply:
(There's others to update too if you do)

Reply:
Yeah I'll fix. thanks for catching